src/ring_theory/ideal/operations.lean

```diff
@@ -2401,4 +2401,110 @@ lemma quot_quot_equiv_comm_algebra_map (x : R) :
 
 end algebra
 
+section algebra_quotient
+
+variables (R) (A : Type v) [comm_semiring R] [comm_ring A] [algebra R A]
+
+variables (I J : ideal A)
+
+/-- The natural algebra homomorphism `A / I → A / (I ⊔ J)`. -/
+def quot_left_to_quot_supₐ : A ⧸ I →ₐ[R] A ⧸ (I ⊔ J) :=
+alg_hom.mk (quot_left_to_quot_sup I J) rfl (map_mul _) rfl (map_add _) (λ _, rfl)
```

Comment:
I recommend adding two `simp` lemmas:
* `(quot_left_to_quot_supₐ R I J).to_ring_hom = quot_left_to_quot_sup I J` (proved by `rfl`)
* `\u=(quot_left_to_quot_supₐ R I J) = quot_left_to_quot_sup I J` (proved by `rfl`)

Reply:
Same for each of the `alg_hom`s below